### PR TITLE
Implement Video Modal, Landing Page Image, 404 page, Copy Link in Cryptobot single

### PR DESCRIPTION
**Pull request by harsh242**
**What does this PR do?**
Implements Video Modal, Landing Page Image, 404 page, Copy Link in Cryptobot single, Sharable Link in '\transaction' & '\claim-transaction'

### CODE REVIEW

**Comment by harsh242** [comment]
@bhaskarSingh  please review this PR.

File: src/pages/tezos/claim-transaction.js [hunk 1]
```diff
@@ -152,8 +153,17 @@ function Transaction({ location }) {
   const [user, setUser] = useAtom(userAtom);
   const [isUser] = useAtom(isUserAtom);
 
+  const [botTokenId, setTokenId] = useState(false);
+  const [copyLink, setCopyLink] = useState(false);
   const [claimButtonDisabled, setClaimButtonDisabledStatus] = useState(true);
 
+  useEffect(() => {
```
Comment (harsh242) [inline src/pages/tezos/claim-transaction.js:160]: Also is there's any better way to redirect, bot doesn't exist?

File: src/pages/tezos/transaction.js [hunk 2]
```diff
@@ -136,12 +137,21 @@ function Transaction({ location }) {
...
   const [user, setUser] = useAtom(userAtom);
 
   const [claimButtonDisabled, setClaimButtonDisabledStatus] = useState(true);
 
+  useEffect(() => {
+    if (bot===false) {
+      console.log(window)
+      window.location.pathname="/tezos/marketplace";
+    }
+  }, []);
```
Comment (harsh242) [inline src/pages/tezos/transaction.js:153]: Essentially I am redirecting user to the marketplace if the bot doesn't exist, it restricts user to directly g o to URL `/tezos/transaction` and see an empty screen.
Any better way to redirect?

File: src/pages/tezos/claim-transaction.js (hunk 1 above)
Comment (bhaskarSingh) [reply src/pages/tezos/claim-transaction.js:160]: I think redirecting users here will confuse them. It's better to show an error & a back button to go to the previous page.

File: src/pages/tezos/index.js [hunk 3]
```diff
@@ -76,14 +77,58 @@ const TestimonialCard = ({ link, img, name, username, text }) => {
...
+          onClick={() => setVideoModal(0)}
+          className="h-12 w-12 rounded-full bg-base-500 absolute right-8 top-8 cursor-pointer flex items-center justify-center"
+        >
+          <MdClose size="24px" />
+        </div>
+        <div>
+          <iframe
+            width="800"
+            height="450"
+            src="https://www.youtube.com/embed/8VrlUL8W__Q?autoplay=1"
```
Comment (bhaskarSingh) [inline src/pages/tezos/index.js:97]: This is the link to phase 1 of the project. Needs to be replaced with the phase 2 video.

File: src/pages/tezos/transaction.js (hunk 2 above)
Comment (bhaskarSingh) [reply src/pages/tezos/transaction.js:153]: I think redirecting users here will confuse them. It's better to show an error & a back button to go to the previous page.

**Pull request merged**